docker/readme.md

```diff
@@ -0,0 +1,45 @@
+# Docker
+
+Docker is an amazing container environment tool that can be used to normalize development and production environments.
+
+## Installing Docker
+
+Installing docker on different operating systems:
+- Mac: [Docker Installation for Mac](https://docs.docker.com/docker-for-mac/install/)
```

Comment:
Most of us use Homebrew. Can you include a link to get it brew installed?

Reply:
Indeed, let me get that started.